EventProc/python/overlay.py

```diff
@@ -0,0 +1,38 @@
+"""Configuration for overlay """
+
+
+
+from LDMX.Framework import ldmxcfg
+
+class OverlayProducer(ldmxcfg.Producer) :
+    """Configuration for pileup overlay
+
+        Sets all parameters to reasonable defaults.
```

Comment:
Document your parameters here as well as in the Cpp source. General format:
```
Parameters
----------------
Name : type
     Description

Attributes:
-------------
name : type
    Description
```
Parameters are for the inputs to the constructor. Attributes are for everything else.